iceoryx2-cli/utils/src/format.rs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2024 Contributors to the Eclipse Foundation
+//
+// See the NOTICE file(s) distributed with this work for additional
+// information regarding copyright ownership.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Apache Software License 2.0 which is available at
+// https://www.apache.org/licenses/LICENSE-2.0, or the MIT license
+// which is available at https://opensource.org/licenses/MIT.
+//
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+
+use anyhow::{anyhow, Context, Error, Result};
+use clap::ValueEnum;
+use serde::Serialize;
+use std::str::FromStr;
+
+#[derive(Clone, Copy, ValueEnum)]
+#[value(rename_all = "UPPERCASE")]
+pub enum Format {
```

Comment:
This is really nice that we can expand the CLI quickly to support also other formats.